ts/input/tex/NodeUtil.ts

```diff
@@ -33,62 +33,62 @@ import { Property, PropertyList } from '../../core/Tree/Node.js';
 import { Args } from './Types.js';
 import { OperatorDef } from '../../core/MmlTree/OperatorDictionary.js';
 
-namespace NodeUtil {
-  const attrs: Map<string, boolean> = new Map([
-    ['autoOP', true],
-    ['fnOP', true],
-    ['movesupsub', true],
-    ['subsupOK', true],
-    ['texprimestyle', true],
-    ['useHeight', true],
-    ['variantForm', true],
-    ['withDelims', true],
-    ['mathaccent', true],
-    ['open', true],
-    ['close', true],
-  ]);
+const attrs: Map<string, boolean> = new Map([
+  ['autoOP', true],
+  ['fnOP', true],
+  ['movesupsub', true],
+  ['subsupOK', true],
+  ['texprimestyle', true],
+  ['useHeight', true],
+  ['variantForm', true],
+  ['withDelims', true],
+  ['mathaccent', true],
+  ['open', true],
+  ['close', true],
+]);
```

Comment:
It looks like this could be made into a `Set` rather than a `Map`.  Also, should this be a proper of the `NodeUtil` object so that the list can be augmented if needed?

Reply:
Done. Ultimately, all these strings should become string enums, so we don't run into problems with expressions like `{open: true, stretchy: true}` etc. when we are using different bundlers (rollup, tscc).